### ISSUE TITLE
mastodon.account_search not working correctly?

### ISSUE THREAD

**Issue by pdeitel** [halcy/Mastodon.py#361]
When I try
`accounts = mastodon.account_search(q='Mastodon') `
I get back mostly results that don't contain the word "Mastodon" in the account name and the main Mastodon account is not even included in the 40 results that come back. What am I missing?

**Comment by craigmaloney** [comment]
Which version of Mastodon.py are you using? I'm running a similar query and it appears to be returning the correct results. This is on octodon.social with Mastodon.py 1.8.1.

**Comment by pdeitel** [comment]
Exact same version.

**Comment by pdeitel** [comment]
I am logged into my account through mastodon.social though I don't think that query requires it.

**Comment by craigmaloney** [comment]
Could you try a different server to see if you have the same results? I tried on another server and can't seem to replicate the same results you're experiencing. The search is reliant on whatever the server returns.

**Comment by pdeitel** [comment]
Hmmm. So I switched to mastodon.online and used a separate account I have there. Doing the same search did in fact return the main mastodon account on mastodon.social. But it's also returning mostly account names that don't include the word "mastodon." 

When I first started experimenting with Mastodon.py earlier this year, this same search would only return accounts with names that included the word I was searching for. strange.

**Comment by craigmaloney** [comment]
Yeah, that is weird. Wonder if a note to Mastodon.social to ask what's going on is in order?

**Comment by pdeitel** [comment]
I'll do that.

**Comment by halcy** [comment]
Closing this as it seems to be more of a weird server specific thing rather than an api impl bug

**Comment by pdeitel** [comment]
OK. But is it strange that most of the results coming back from multiple servers don't have the search term in the user name?

**Comment by halcy** [comment]
It is slightly weird, tbh. Can you post a short repro? I will check it out

**Comment by halcy** [comment]
closing since I unfortunately cannot reproduce
